include/seqan/sequence/adapt_stl_container.h

```diff
@@ -585,6 +585,10 @@ capacity(std::array<TChar, N> const & me)
 // Function empty
 // ----------------------------------------------------------------------------
 
+// VC2015 implements some C++17 functions which would collide for 
+// applications that do using namespace std
+// http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2014/n4280.pdf
+#if _MSC_VER < 1900
```

Comment:
I think that VS2015 does not imply any of C++11/14/17.
You should be able to instruct VC to respect the C++11 (or 14) standard, like the flag -std=... for gcc, right?
I don't have/use VC... maybe this helps: https://msdn.microsoft.com/en-us/library/jj851139.aspx.